### PR TITLE
clearing last of ui fixes, implementing a mobile message

### PR DESCRIPTION
**Pull request by loganjross**

### CODE REVIEW

File: app/src/components/modals/VoteModal.tsx [hunk 1]
```diff
@@ -93,9 +91,6 @@ export const VoteModal = ({
         undefined,
         voteRecord ? voteRecord!.pubkey : undefined
       )
-        .then(() => {
-          setCurrent(Steps.VoteSuccess);
-        })
```
Comment (Tristyn) [inline app/src/components/modals/VoteModal.tsx:98]: So once you cast a vote nothing happens? Not saying it's successful not even closing the modal?

**Review by Tristyn — CHANGES_REQUESTED** [review]
Pls clarify if doing nothing after casting a vote is intentional

File: app/src/components/modals/VoteModal.tsx (hunk 1 above)
Comment (Tristyn) [reply app/src/components/modals/VoteModal.tsx:98]: Logan and I chatted and after a successful transaction he'll close the modal with `onClose` for all the modals

**Review by callensm — COMMENTED** [review]
Somethings weird about this PR. It shouldn't be containing > 10 arbitrary duplicate commits from previous merges.

**Comment by loganjross** [comment]
I will take a look, i must have merged master on accident

**Pull request closed without merging**